doc/user_guide/model.rst

```diff
@@ -716,9 +746,9 @@ by hand.
     :py:meth:`~.model.Model.enable_adjust_position` and
     :py:meth:`~.model.Model.disable_adjust_position`
 
-Also, :py:meth:`~.model.Model.enable_adjust_position` provides an interactive
+Also, :py:meth:`~.models.model1D.Model1D.enable_adjust_position` provides an interactive
 way of setting the position of the components with a well define position.
-:py:meth:`~.model.Model.disable_adjust_position` disables the tool.
+:py:meth:`~.models.model1D.Model1D.disable_adjust_position` disables the tool.
```

Comment:
Sure - I think that was a relic from before I also included these methods in Model2D with the NotImplementedError - will change.